src/JSONLD/JSONLD_DOI.jl

```diff
@@ -0,0 +1,10 @@
+struct JSONLD_DOI <: JSONLD end
+
+function mainpage_url(repo::JSONLD_DOI, dataname)
+    if match_doi(dataname) != nothing
+        url = joinpath("https://data.datacite.org/", match_doi(dataname))
+        resp = HTTP.get(url, ["Accept"=>"application/vnd.schemaorg.ld+json"]; forwardheaders=true)
+        json = JSON.parse(resp.body |> String |> strip)
```

Comment:
If you are going to use the pipe operator, use it completely, or not at all.
`resp.body |> String |> strip) |> JSON.parse`

Or

`JSON.parse(strip(String(resp.body)))`
Also, is the `strip` required?